app-admin/pwman3/pwman3-0.12.2.ebuild

```diff
@@ -24,8 +25,13 @@ RDEPEND="
 "
 BDEPEND="test? ( dev-python/pexpect[${PYTHON_USEDEP}] )"
 
+
 distutils_enable_tests setup.py
```

Comment:
This also needs to actually be removed...

Reply:
oops ...